### PR TITLE
Convert local fixture config

### PR DESCRIPTION
**Pull request by noreklint**
Converts `fixture.config` to `fixture.json`

```
som-converter_1  | 2022-01-24 14:13:46 -- INFO	**Converter** Converting all local fixture config in /projects
som-converter_1  | 2022-01-24 14:13:46 -- INFO	**Converter** Converting /projects/PayamMartin19Oct16Scanner1/fixture.config (1 entries)
som-converter_1  | 2022-01-24 14:13:46 -- INFO	**Converter** Converted 1 local fixture config files
```
File looks like
```
{
  "__MODEL__": "FixtureModel",
  "__CONTENT__": {
    "scale": 1,
    "name": "Delta 1",
    "coordinates_scale": 1,
    "grayscale": {
      "__MODEL__": "GrayScaleAreaModel",
      "__CONTENT__": {
        "x1": 290,
        "name": "SilverFast",
        "x2": 410,
        "width": -1,
        "section_length": -1,
        "values": [
          21.5,
          24.0069444444,
          28.5075757576,
          32.9097222222,
          39,
          46.0378787879,
          52.0069444444,
          59.946969697,
          70.1458333333,
          79.9545454545,
          88.1875,
          100.277777778,
          110.522727273,
          122.236111111,
          135.037878788,
          147.222222222,
          160.106060606,
          171.444444444,
          184.386363636,
          195.6875,
          212.444444444,
          224.513888889,
          227.284722222
        ],
        "y1": 2540,
        "y2": 3300
      }
    },
    "shape": [
      6000,
      4800
    ],
    "orientation_marks_y": [
      3008.90409062,
      216.785702076,
      5801.06627029
    ],
    "orientation_marks_x": [
      2688.07670601,
      307.686217216,
      342.87715987
    ],
    "plates": [
      {
        "__MODEL__": "FixturePlateModel",
        "__CONTENT__": {
          "y1": 0,
          "index": 1,
          "x1": 2830,
          "y2": 2850,
          "x2": 4760
        }
      },
      {
        "__MODEL__": "FixturePlateModel",
        "__CONTENT__": {
          "y1": 20,
          "index": 2,
          "x1": 530,
          "y2": 2870,
          "x2": 2550
        }
      },
      {
        "__MODEL__": "FixturePlateModel",
        "__CONTENT__": {
          "y1": 3120,
          "index": 3,
          "x1": 2850,
          "y2": 5980,
          "x2": 4770
        }
      },
      {
        "__MODEL__": "FixturePlateModel",
        "__CONTENT__": {
          "y1": 3130,
          "index": 4,
          "x1": 540,
          "y2": 5960,
          "x2": 2510
        }
      }
    ],
    "path": "/home/jw20/.scan-o-matic/config/fixtures/delta_1.config",
    "orentation_mark_path": ""
  }
}
```

@local-minimum What is `path` for? Should it be handled in some way?

### CODE REVIEW

File: converter.py [hunk 1]
```diff
@@ -50,7 +51,7 @@ def get_dump_name(filename, extension):
 )
 
 for file_type, pattern, loader, processor, new_ext in (
-    (
+     (
```
Comment (skymandr) [inline converter.py]:
```suggestion
    (
```

File: converter.py [hunk 2]
```diff
@@ -105,3 +113,5 @@ def get_dump_name(filename, extension):
 
         n += 1
     logger.info('Converted {} {} files'.format(n, file_type))
+
+
```
Comment (skymandr) [inline converter.py]:
```suggestion
```

**Review by skymandr — APPROVED** [review]
The two minor comments aside, it looks good to me, but probably best to wait for @local-minimum, especially since you had a question.

File: converter.py [hunk 3]
```diff
@@ -73,7 +74,14 @@ def get_dump_name(filename, extension):
...
         },
         ".json",
-    )
+    ),
+    (
+        'local fixture config',
+        paths.experiment_local_fixturename,
+        FixtureFactory.serializer.load_first,
+        None,
+        ".json",
```
Comment (local-minimum) [inline converter.py]: I think it's better to write it back into the file
```suggestion
        None,
```
The reason @skymandr needed name change was that it was called `.npy` before which most def not a json, but `.fixture` could be anything.

**Review by local-minimum — CHANGES_REQUESTED** [review]

**Comment by local-minimum** [comment]
path is where the file was originally located and not that useful and there shouldn't be any need to handle it. Most path things in scanomatic has fallback behavor so that it attempts to find things with alternative methods if the info in path fails. But here I doubt it's useful at all.

**Review by local-minimum — APPROVED** [review]

**Pull request merged**